cpp/example_code/dynamodb/Readme.md

```diff
@@ -1,17 +1,25 @@
 # Amazon DynamoDb code examples for the SDK for C++
+
 ## Overview
+
 Sample code which demonstrates creation, deletion, modification, and querying of Amazon DynamoDB databases.
 
 *Amazon DynamoDB is a fully managed NoSQL database service that provides fast and
 predictable performance with seamless scalability.*
+
 ## ⚠️ Important
+* 
 * Running this code might result in charges to your AWS account.
 * Running the tests might result in charges to your AWS account
 * We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
 * This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
 ## Code examples
+
 ### Single actions
+
 Code excerpts that show you how to call individual service functions.
+
 * [Get items in a batch](./batch_get_item.cpp) (BatchGetItem)
```

Comment:
What about the PartiQL single actions, should those be added for C++ also?

Reply:
Two scenarios for one issue.  I did not realize.  I should get more story points,.